packages/sources/react/src/components/VtmnToggle/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as VtmnToggle } from './VtmnToggle';
+export * from './VtmnToggle';
```

Comment:
Can't these two lines be merged ?

Reply:
No i think we have to keep it like that, it's done like that for others components.
@lauthieb what's your advice on it ?

Reply:
Yep same we can't :/